src/Lykke.Job.LiteCoin/Startup.cs

```diff
@@ -124,7 +125,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, IApplica
         {
             try
             {
-                // NOTE: Job not yet recieve and process IsAlive requests here
+                ApplicationContainer.Resolve<PeriodicalHandlerHost>().Start();
```

Comment:
This is minor thing, but I believe, will be better to inject PeriodicalHandlerHost to the StartupManager and start it there